agent/internal/sigmalr/sigmalr.go

```diff
@@ -63,10 +64,19 @@ func (g *GinMappable) HandleFunc(pattern string, handler func(writer http.Respon
 }
 
 func Log(requestID *string, messages ...string) {
-	if requestID != nil && SignalrServ != nil {
+	serv, err := GetSignalrInstance()
+	if err != nil {
+		panic(err)
```

Comment:
dont panic. panic feels bad in a log, is it necessary?

Reply:
Are you sure? ![image](https://user-images.githubusercontent.com/9247788/181252438-641a800e-bc40-4a2f-a13a-11872cf4b2fc.png)

Reply:
but this is a log, in some cases it is okay to panic. Review all the panics -> could be a good ticket.